src/collective/blueline/browser/viewlets.py

```diff
@@ -31,6 +34,25 @@ def show(self):
         show = api.user.is_anonymous() or self.show_authenticated
         return not is_configlet and show
 
+    def evaluate_code(self, code):
```

Comment:
don't work
    ExpressionError('invalid syntax', 'context/@@plone_portal_state/portal_url') (Also, the following error occurred while attempting to render the standard error message, please see the event log for full details: ('invalid syntax', u'context/@@plone_portal_state/portal_url'))